.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!-- Thank you for contributing to our WordPress open source project! -->
+
+## Description
+<!-- Please provide a clear and concise description of your changes -->
+<!-- What problem does this PR solve? What functionality does it add/improve? -->
+
+## Related Issue
+<!-- Please link any related issues here. Use the syntax: "Fixes #123" or "Related to #456" -->
+
+## Dependant PRs
+<!-- List any dependent PR's that are awaiting review. Use the following syntax: "#1234" -->
+
+## Type of Change
+<!-- What types of changes does your code introduce? Put an x in all boxes that apply -->
+- [ ] ✅ Bug fix (non-breaking change which fixes an issue)
+- [ ] ✨ New feature (non-breaking change which adds functionality)
+- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] 🧹 Code refactoring (no functional changes)
+- [ ] 📄 Example update (no functional changes)
+- [ ] 📝 Documentation update
+- [ ] 🔍 Performance improvement
+- [ ] 🧪 Test update
+
+## How Has This Been Tested?
+<!-- Please describe the tests you've run to verify your changes -->
+<!-- Include details of your testing environment, and the tests you ran -->
+
+## Screenshots
+<!-- If your change affects the UI or UX, provide before/after screenshots or videos -->
+<!-- Delete this section if not applicable -->
+
+## Checklist
+<!-- Put an x in all the boxes that apply -->
+- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) document
+- [ ] My code follows the project's coding standards
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation (if applicable)
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
+- [ ] Any dependent changes have been highlighted, merged or published
+
+## Additional Context
+<!-- Add any other context about the PR here. -->
```

Comment:
It feels unnecessary; description is open-ended enough...and people know to add more if they think it's important